index.html

```diff
@@ -1,13 +1,76 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="style.css">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="style.css" />
     <title>Document</title>
-</head>
-<body>
-   <h1 class="color-blue">Hello Microverse</h1> 
-</body>
-</html>
\ No newline at end of file
+  </head>
+
+  <body>
+    <header>
+      <nav>
+        <div class="head">
+          <div class="logo">
+            <p>My logo</p>
+          </div>
+          <div class="ham">
+            <i class="fa">&#xf0c9;</i>
+          </div>
+        </div>
+      </nav>
+    </header>
+
+    <!-- HOME PAGE-->
+    <main class="main">
+      <div class="main-content">
+        <div class="author">
+          <h1>
+            I'm Imo. <br />
+            Glad to see <br />
+            you!
+          </h1>
+        </div>
+        <div class="content">
+          <p>
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hesitate to
+            contact me.
+          </p>
+        </div>
+      </div>
+
+      <div class="socials">
+        <div class="connect">
+          <a href="#" class="link">LETS CONNECT</a>
+        </div>
```

Comment:
- [OPTIONAL] I highly recommend wrapping this sub-title `LET'S CONNECT` into a heading tag `for example h2,h3,...` instead of the Anchor `a` tag. because it's only a title and it doesn't direct us to any part of the page or any other website. I think it's better to use a heading tag for following the best HTML best practices.